dist/jowl.spec

```diff
@@ -0,0 +1,88 @@
+Name:           jowl
+Version:        2.0.2
+Release:        1%{?dist}
+Summary:        CLI for JSON operations with Lodash
+
+License:        MIT
+URL:            https://jowl.app
+Source0:        https://github.com/daxelrod/%{name}/archive/v%{version}/%{name}-%{version}.tar.gz
+# Also at       https://registry.npmjs.org/{name}/-/{name}-{version}.tgz
+# however, npmjs.org has a build product which does not contain docs or tests
+
+# The following sources are generated by running
+# nodejs-packaging-bundler against Source0:
+Source1:        %{name}-%{version}-nm-prod.tgz
+Source2:        %{name}-%{version}-nm-dev.tgz
+Source3:        %{name}-%{version}-bundled-licenses.txt
```

Comment:
Add the script that generates these sources as a Source so it's included in the SRPM.

Reply:
The script that generates these sources is just nodejs-packaging-bundler, which is already packaged in Fedora.  I've attempted to make the comments in packit-fix-spec-file.sh clearer that it's only necessary to run nodejs-packaging-bundler to generate sources, and the rest is all packit-specific stuff that is unecessary in a normal RPM build.